starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,13 +1,22 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="css/main.css">
   <title>Apple Pie Recipe</title>
 </head>
+
+<img src="images/apple-pie.jpg" alt="">
```

Comment:
Maybe you need the image to be the background of your title. It could be defined as:

```
// HTML
<div class="title">
   <h1>
      Apple Pie
   </h1>
</div>

//CSS
.title{
   background-image: url("../images/your-image.png");
}
```
The background will be conditioned by the div size (because it's a background). Look for more information about backgrounds and backgroud styling in here:

https://developer.mozilla.org/en-US/docs/Web/CSS/background-image